core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -207,6 +209,8 @@ public ServerBuilder() {
         virtualHostTemplate.accessLogger(
                     host -> LoggerFactory.getLogger(defaultAccessLoggerName(host.hostnamePattern())));
         virtualHostTemplate.tlsSelfSigned(false);
+        virtualHostTemplate.annotatedHttpServiceExtensions(ImmutableList.of(), ImmutableList.of(),
```

Comment:
How about not setting `annotatedHttpServiceExtensions` here so that we do not call `buildServiceConfigBuilder` in `ServerBuilder.build()` when the `extenstions` is null?

Reply:
`buildServiceConfigBuilder` should always be called. So I set a default value to reduce the complexity of `buildServiceConfigBuilder`.

Am I right about your question?

Reply:
Ah, yes. I misread it. So it's okay as it is. 😉